Source/Editor/Gizmo/TransformGizmoBase.Settings.cs

```diff
@@ -7,56 +7,51 @@ namespace FlaxEditor.Gizmo
-        };
-
-        private BoundingBox XAxisBox = new BoundingBox(new Vector3(LineOffset, -SingleAxisThickness, -SingleAxisThickness), new Vector3(LineOffset + LineLength, SingleAxisThickness, SingleAxisThickness));
-        private BoundingBox YAxisBox = new BoundingBox(new Vector3(-SingleAxisThickness, LineOffset, -SingleAxisThickness), new Vector3(SingleAxisThickness, LineOffset + LineLength, SingleAxisThickness));
-        private BoundingBox ZAxisBox = new BoundingBox(new Vector3(-SingleAxisThickness, -SingleAxisThickness, LineOffset), new Vector3(SingleAxisThickness, SingleAxisThickness, LineOffset + LineLength));
-        private BoundingBox XZBox = new BoundingBox(Vector3.Zero, new Vector3(LineOffset, MultiAxisThickness, LineOffset));
-        private BoundingBox XYBox = new BoundingBox(Vector3.Zero, new Vector3(LineOffset, LineOffset, MultiAxisThickness));
-        private BoundingBox YZBox = new BoundingBox(Vector3.Zero, new Vector3(MultiAxisThickness, LineOffset, LineOffset));
-        private BoundingBox CenterBoxRaw = new BoundingBox(new Vector3(-0.5f * CenterBoxSize), new Vector3(0.5f * CenterBoxSize));
-        private float RotateRadiusRaw = 4.0f;
-
-        private BoundingSphere ScaleXSphere => new BoundingSphere(Vector3.Transform(_translationLineVertices[1], _gizmoWorld), ScaleSpheresRadius * _screenScale);
-        private BoundingSphere ScaleYSphere => new BoundingSphere(Vector3.Transform(_translationLineVertices[7], _gizmoWorld), ScaleSpheresRadius * _screenScale);
-        private BoundingSphere ScaleZSphere => new BoundingSphere(Vector3.Transform(_translationLineVertices[13], _gizmoWorld), ScaleSpheresRadius * _screenScale);
-        private OrientedBoundingBox CenterBox => new OrientedBoundingBox(CenterBoxRaw) * _gizmoWorld;
+        /// <summary>
+        /// Scale of the gizmo itself
+        /// </summary>
+        private const float GizmoScaleFactor = 24;
+
+        /// <summary>
+        /// The length of each axis (outwards)
+        /// </summary>
+        private const float AxisLength = 3.5f;
+
+        /// <summary>
+        /// Offset to move axis away from center
+        /// </summary>
+        private const float AxisOffset = 0.8f;
+
+        /// <summary>
+        /// How thick the axis should be
+        /// </summary>
+        private const float AxisThickness = 0.3f;
+
+        /// <summary>
+        /// Center box scale
+        /// </summary>
+        private const float CenterBoxScale = 0.8f;
+
+        /// <summary>
+        /// The inner minimum of the multiscale
+        /// </summary>
+        private const float InnerExtend = AxisOffset + 0.5f;
+
+        /// <summary>
+        /// The outer maximum of the multiscale
+        /// </summary>
+        private const float OuterExtend = AxisOffset * 3.5f;
+
+        private BoundingBox XAxisBox = new BoundingBox(new Vector3(-AxisThickness), new Vector3(AxisThickness)).MakeOffsetted(AxisOffset * Vector3.UnitX).Merge(AxisLength * Vector3.UnitX);
```

Comment:
For those wondering what this does, it creates a cube with the size `AxisThickness`, then moves it along the axis (`AxisThickness`) and finally makes it really long (`AxisLength`)